Automattic/woocommerce-payments
# Use shared dispute utility function `isRefundable` for disputed order notice to improve code quality

#### Changes proposed in this Pull Request

This PR utilises the shared dispute util function `isRefundable` to determine if a disputed order is refundable on the order screen.

https://github.com/Automattic/woocommerce-payments/blob/ea14b50d3314726ec5a15e102d76ecc5a4b0990c/client/disputes/utils.ts#L75-L78

#### Testing instructions

**Note: PR author has confirmed that the following flows are working as expected**

* Create a disputed order using the card `4000000000000259` at checkout.
* View the order and ensure that the refund button **is disabled**.
* "Win" the dispute by challenging the dispute with `winning_evidence` in the "additional details" field.
* View the order and ensure that the refund button **is not disabled**.
* Create an inquiry on an order using the card `4000000000001976` at checkout.
* View the order and ensure that the refund button **is not disabled**.

-------------------


**Post merge**
